Documentation/gettingstarted/ipvlan.rst

```diff
@@ -107,20 +101,6 @@ masquerading all traffic leaving the node:
      --set global.masquerade=true \\
      --set global.autoDirectNodeRoutes=true
 
-Example ConfigMap extract for ipvlan in L3 mode with more efficient
-BPF-based masquerading instead of iptables-based:
-
-.. parsed-literal::
-
-   helm install cilium |CHART_RELEASE| \\
-     --namespace kube-system \\
-     --set global.datapathMode=ipvlan \\
-     --set global.ipvlan.masterDevice=bond0 \\
-     --set global.tunnel=disabled \\
-     --set global.masquerade=true \\
-     --set global.installIptablesRules=false \\
-     --set global.autoDirectNodeRoutes=true
-
```

Comment:
Do you plan to re-add the documentation here on how to use ipvlan in L3-only mode with the ip masq agent?

Reply:
Yep, but only after we have tested whether the ipvlan masq actually works.